### PR TITLE
LRQA-80465 | master

### PR DESCRIPTION
**Pull request by allen-nagtalon**
https://issues.liferay.com/browse/LRQA-80465

### CODE REVIEW

**Comment by allen-nagtalon** [comment]
ci:test

**Comment by allen-nagtalon** [comment]
ci:test

**Comment by allen-nagtalon** [comment]
ci:test

**Comment by allen-nagtalon** [comment]
ci:test

File: portal-web/test/functional/com/liferay/portalweb/macros/liferayfaces/PortalShowcase.macro [hunk 1]
```diff
@@ -3,51 +3,85 @@ definition {
...
 
-		RobotType.robotTypeShortcut(locator1 = "Ctrl + b");
+		Type.sendKeys(
+			locator1 = "PortalShowcase#WRITEABLE_FIELD",
+			value1 = "keys=CONTROL,b");
 
-		RobotType(locator1 = "lo t");
+		Type(
+			locator1 = "PortalShowcase#WRITEABLE_FIELD",
+			value1 = "lo to");
```
Comment (CalumR23) [inline portal-web/test/functional/com/liferay/portalweb/macros/liferayfaces/PortalShowcase.macro:20]: an extra "o" here

**Pull request closed without merging**